### ISSUE TITLE
Add Red Green and Blue colourmaps

### ISSUE THREAD

**Issue by PolarBean** [FZJ-INM1-BDA/siibra-explorer#1366]
A common way to visualise single channel data is to map it into either red green or blue channels. The relevant neuroglancer shaders to achieve this are below. It could be a very useful feature to add these three to the currently available colourmaps. 

Thank you in advance

Blue

```
#uicontrol invlerp normalized
void main () {
  emitRGB(vec3(toNormalized(getDataValue(1)),
               toNormalized(getDataValue(2)),
              normalized()));
}
```

Red
```
#uicontrol invlerp normalized
void main () {
  emitRGB(vec3(normalized(),
               toNormalized(getDataValue(1)),
               toNormalized(getDataValue(2))));
}
```

Green
```
#uicontrol invlerp normalized
void main () {
  emitRGB(toNormalized(getDataValue(1)),
              normalized(),
               toNormalized(getDataValue(2))));
}
```

**Comment by xgui3783** [comment]
Do you mean that some of your volumes are single channeled RGB?

Adding the colormaps are easy enough. Greyscale is done thusly so:

https://github.com/xgui3783/ng-layer-tune/blob/master/src/utils/colormaps.ts#L205-L209

We can rather easily add a RED/GREEN/BLUE inaddition to GREY.

But I don't think compositing three single channel volumes will result in something similar to a three channelled RGB volume, since we don't control how the layers blends.

**Comment by xgui3783** [comment]
hmm, overlaying multiple volumes is... possible, but certainly not a priority for the time being. 

We will try to add RED/GREEN/BLUE options for the shader.

**Comment by PolarBean** [comment]
haha I agree completely that its not a priority. Thank you for adding the color options :)

**Comment by xgui3783** [comment]
will be fixed by #1371 

(fixed by #1370)
